shared/fs/sync-status/container.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+
```

Comment:
🧨

Reply:
I don't have that emoji yet!

Reply:
Sounds like you needed a macOS upgrade 😉